src/website/interface.ts

```diff
@@ -4,6 +4,7 @@ import { ChallongeIDConflictError, UserError } from "../util/errors";
 export interface WebsiteWrapper {
 	createTournament(name: string, desc: string, url: string, topCut?: boolean): Promise<WebsiteTournament>;
 	updateTournament(tournamentId: string, name: string, desc: string): Promise<void>;
+	updateTieBreakers(tournamentId: string, tbs: challongeTieBreaker[]): Promise<void>;
```

Comment:
A stub has to be in the mock thing too, even if we're not writing tests for this.